absearch/server.py

```diff
@@ -37,6 +38,34 @@ def reload():
 gevent.signal(signal.SIGINT, close)
 
 
+class _Statsd(object):
+    def __init__(self, config):
+        if config.get('datadog', True):
```

Comment:
is the datadog client compatible with statsd ?

if not that also a change you'll want to make explicit in changelog